src/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "sajari/sajari-sdk-php",
+  "description": "Sajari SDK for PHP",
+  "minimum-stability": "dev",
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/stanley-cheung/Protobuf-PHP"
+    }
+  ],
+  "require": {
+    "grpc/grpc": "dev-release-0_14"
```

Comment:
@tbillington you can put `php` in the require too so that the min PHP version is specified. Since you use array notation `[]` which is new in PHP 5.5 (I think) you might want to specify the min version. Or, at least make sure it's specified in the README.md